src/CONST.js

```diff
@@ -344,6 +344,8 @@ const CONST = {
         IMAGE: 'image',
     },
 
+    ADD_PAYMENT_MENU_POSITION_Y: 226,
+    ADD_PAYMENT_MENU_POSITION_X: 356,
```

Comment:
Do we want to work with fixed values here independent of the platform and screen size? Just to be sure this will be how we're imagining in every device.

Reply:
This is a bit of temp code and a follow up created to address this [here](https://github.com/Expensify/Expensify/issues/191203)